cmd/root.go

```diff
@@ -31,12 +32,8 @@ var cfgFile string
 var rootCmd = &cobra.Command{
 	Use:   "onelogin",
 	Short: "A CLI for managing IAM and Authentication resources",
-	Long: `A longer description that spans multiple lines and likely contains
-examples and usage of using your application. For example:
-
-Cobra is a CLI library for Go that empowers applications.
-This application is a tool to generate the needed files
-to quickly create a Cobra application.`,
+	Long: `The OneLogin CLI provides a convenient interface for managing OneLogin resources
+	from the command line such as Apps and User Mappings. `,
```

Comment:
I believe this indentation at the beginning of line 36 will be included in the total string:
https://stackoverflow.com/questions/28113427/indentation-style-for-multi-line-string-literals?lq=1

Reply:
you're right it does :( will fix